contents/docs/surveys/implementing-custom-surveys.mdx

```diff
@@ -8,28 +8,69 @@ availability:
+At its most basic level, a survey is a collection of response events. If you wanted to, you could capture events from anywhere, like in this example of a hardcoded Next.js feedback survey:
 
-If you're using PostHog's [survey API](/docs/surveys/creating-surveys#presentation) to create a custom survey UI, there are 2 steps to integrating with PostHog:
+```js-web
+'use client'
+import { usePostHog } from 'posthog-js/react';
+
+export default function Feedback() {
+  const posthog = usePostHog();
+
+  const handleFeedbackSubmit = (e) => {
+    e.preventDefault();
+    const feedback = e.target.elements.feedback.value;
+    posthog.capture("survey sent", {
+      $survey_id: '018ed910-3b24-0000-e3fd-d51c59aa74b2',
+      $survey_response: feedback
+    })
+  }
+
+  return (
+    <div>
+      <h1>Give us feedback</h1>
+      <form onSubmit={handleFeedbackSubmit}>
+        <textarea 
+          id="feedbackInput" 
+          name="feedback" 
+          placeholder="Enter your feedback here..." 
+          required
+        ></textarea>
+        <button type="submit">Submit Feedback</button>
+      </form>
+    </div>
+  );
+}
+```
+
+The benefit of using PostHog beyond this is that it handles:
+
+1. **Survey content.** Customize question type, text, and more. Change it at runtime without needing to redeploy your app.
+
+2. **Targeting logic.** This means not showing the same survey multiple times, the wrong survey, or a survey that has collected enough responses. Leverage property filters, cohorts, feature flags, and more.
 
-## Step 1: Fetch active surveys
+If you create a [popover survey](/docs/surveys/creating-surveys#presentation),  updating and targeting are handled automatically. When you create one in API mode, you need to add logic to fetch and display surveys yourself with the help of the JavaScript Web SDK or snippet.
 
-To fetch the surveys currently enabled for a user, call `posthog.getActiveMatchingSurveys(callback, forceReload)`.
+## Fetching surveys
 
-Alternatively, if you wish to do survey targeting yourself, you can call `posthog.getSurveys(callback, forceReload)` to return a list of all surveys from PostHog.
+When implementing an API survey, there are two options for fetching surveys from PostHog:
 
-Surveys are cached by default in the posthog-js library, so you can pass `true` to `forceReload` to force a new surveys api call from PostHog.
+1. To get all surveys, call `getSurveys(callback, forceReload)`. This means you still need to handle targeting yourself.
 
-Both methods return a callback with an array of `surveys`:
+2. To get surveys enabled for the current user, call `getActiveMatchingSurveys(callback, forceReload)`. This always returns an active survey if the user meets the targeting conditions, even if they have already seen, dismissed, or responded to the survey.
+
+Surveys are cached by default by the JavaScript SDK, so you can pass `true` to the `forceReload` parameter to force an API call to get the latest surveys.
```

Comment:
This makes it sound like I always want to use true. When wouldnt I want to use true? When would I want to use false?

Reply:
```suggestion
Surveys are requested on first load and then cached by default by the JavaScript SDK. If you want to force an API call to get an updated list of surveys, pass `true` to the `forceReload` parameter. You only need to do this if you want changed surveys to appear mid-session for users.
```